docs/release-notes/platform.rst

```diff
@@ -11,6 +11,13 @@ Platform Notes
    * OS's and compilers supported
    * network interconnects supported.
 
+Open MPI uses `PRRTE <https://github.com/openpmix/prrte>`_ as its back-end
```

Comment:
This line and line #43 are somewhat conflicting as they indicate two different meanings. Perhaps the latter section should be removed? I couldn't quite figure out exactly what PRRTE supports by reading their docs though. I'm not sure this is entirely accurate either. I'm not too aware of the details, but I thought PRRTE is used in non scheduled environments. I don't believe that that PRRTE is used by slurm for example (Not 100% certain, though)

Maybe @rhc54 can chime in with what he thinks should go here?

Reply:
> 

Yes, this was confusing.  :-(   I did some more research and updated this paragraph with a clearer (I think! :-) explanation of the idea.